tests/kernel/threads/thread_apis/src/main.c

```diff
@@ -37,7 +37,7 @@ extern void test_k_thread_foreach(void);
 extern void test_threads_cpu_mask(void);
 
 struct k_thread tdata;
-#define STACK_SIZE (256 + CONFIG_TEST_EXTRA_STACKSIZE)
+#define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
CI has been passing (mostly) without this type of changes which also increases the stack size for non-coverage tests. I would recommend defining `CONFIG_TEST_EXTRA_STACKSIZE` in `prj*.conf` instead. We still want to build these with sanitycheck for platforms with smaller memory space.

Reply:
actually I didn't change the original stack size. original logic is that 256 + CONFIG_TEST_EXTRA_STACKSIZE(CONFIG_TEST_EXTRA_STACKSIZE=256 in prj.conf) = 512, my change just remove the defining CONFIG_TEST_EXTRA_STACKSIZE and add every digital number defining stack size by 256.

Reply:
Ah... I see. I missed the change in prj.conf.